coala_quickstart/generation/SettingsClass.py

```diff
@@ -110,21 +151,27 @@ def fillup_optional_settings(self, key, funcs):
             A list of function objects i.e. either containing the run() method
             or the create_arguments() + generate_config() methods of the linter
             bears.
+        :param bear:
+            The bear object.
+        :param settings:
+            Dict of optional bear settings of the form:
+            {'setting_name': default_values,}
```

Comment:
The `default_values` is a list?

If yes you can specify it there.

Reply:
actually I think it will never be unless the type is meant to be a list. There are never going to be multiple default values.